### PR TITLE
mgmt, upgrade AKS 2022-03

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
Live test pass.

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by XiaofeiCao — APPROVED** [review]

File: sdk/resourcemanager/api-specs.json [hunk 1]
```diff
@@ -279,5 +279,11 @@
     "source": "specification/msi/resource-manager/readme.md",
     "package": "com.azure.resourcemanager.msi",
     "args": "--tag=package-2018-11-30 --add-client-flatten=Identity"
+  },
+  "changes": {
```
[the remark this answers is not in the document]
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/api-specs.json:283]: Not needed (I thought this PR will merged while the change RP will be much later, so cherry-picked it, should be no diff after change RP merged).

[the remark this answers is not in the document]
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/api-specs.json]: Refreshed main, this section is gone.

**Pull request merged**